clean_project/search_api_test.py

```diff
@@ -0,0 +1,36 @@
+import json
+import search_api
+import search_engine
+import unittest
+
+from index_test import convert
+
+
+class SearchApiTest(unittest.TestCase):
+    def setUp(self):
```

Comment:
set up is a method that is automatically before every test. you don't need to call it manually, and you don't need to return anything. set everything you will need in the tests as a property of self.